src/api/app/controllers/webui/request_controller.rb

```diff
@@ -345,15 +346,20 @@ def set_filter_state
     @filter_state = @filter_state.intersection(BsRequest::VALID_REQUEST_STATES.map(&:to_s))
   end
 
+  def set_filter_creators
+    @filter_creators = params[:creators].presence || []
+  end
+
   def filter_requests
     @bs_requests = filter_by_text(params[:requests_search_text])
-
     @bs_requests = filter_by_involvement(@bs_requests, @filter_involvement)
     @bs_requests = @bs_requests.where(state: @filter_state) if @filter_state.present?
+    @bs_requests = filter_by_creators(@bs_requests, @filter_creators) if @filter_creators.present?
```

Comment:
```suggestion
    @bs_requests.where(creator: @filter_creators) if @filter_creators.present?
```
I would remove the `filter_by_creators` method. It looks a little bit simplistic to me.

Reply:
I agree

Reply:
done